types/fetch.d.ts

```diff
@@ -0,0 +1,80 @@
+// Type definitions for fetch API
+// Spec: https://fetch.spec.whatwg.org/
+// Polyfill: https://github.com/github/fetch
+// Definitions by: Ryan Graham <https://github.com/ryan-codingintrigue>
+
+interface FetchOptions {
```

Comment:
Is there an `@types/` equivalent for this file?

Reply:
There is, but I can't quite get it to work. I think it might be because of `isomorphic-fetch` and `isomorphic-unfetch` clashing... For now, this feels cleanest